src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -2,17 +2,24 @@
     <aura:handler name="init" value="{!this}" action="{!c.init}"/>
     
     <aura:handler event="c:STG_EVT_Edit" action="{!c.toggleIsView}"/>
+    <aura:handler event="c:STG_EVT_Save" action="{!c.saveAccRecType}"/>
     
-    <aura:attribute name="namespacePrefix" type="String" />
+    <aura:attribute name="accRecTypes" type="Map" />
```

Comment:
Those attributes are being passed from the container component: https://github.com/SalesforceFoundation/HEDAP/blob/feature/216-acc-rec-types-change/src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp#L38